graspy/embed/mds.py

```diff
@@ -120,17 +127,20 @@ def fit(self, X):
 
         Parameters
         ----------
-        X : nd-array
+        X : array_like
             If ``dissimilarity=='precomputed'``, the input should be the 
             dissimilarity matrix with shape (n_samples, n_samples). If 
-            ``dissimilarity=='euclidean'``, then the input should be array 
-            with shape (n_samples, n_features) or a nd-array with shape 
-            (n_samples, n_features_1, n_features_2, ..., n_features_d).
+            ``dissimilarity=='euclidean'``, then the input should be 2d-array 
+            with shape (n_samples, n_features) or a 3d-array with shape 
```

Comment:
clarify that the first case (2d) is operating on the nodes, while the second is operating on graphs